code/modules/antagonists/ert/ert.dm

```diff
@@ -11,6 +11,7 @@
 	var/role = "Security Officer"
 	var/list/name_source
 	var/random_names = TRUE
+	var/rip_and_tear = FALSE
```

Comment:
this should come from the ert team definition I think

Reply:
it shows the fluff text to the ert members separately so i thought that would be a better way to do it